tests/test_environment_guard.py

```
@@ -0,0 +1,136 @@
+            assert result["verified"] is False
+            assert len(result["suspicious_hooks"]) == 1
+            assert len(result["content_findings"]) == 0  # No content scan
+
+    def test_multiple_suspicious_files(self):
+        """Should report all suspicious .pth files, not just the first."""
+        guard = StartupHookGuard()
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            for i in range(3):
+                with open(os.path.join(tmpdir, f"backdoor_{i}.pth"), "w") as f:
+                    f.write(f"import os; os.popen('whoami')\n")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)